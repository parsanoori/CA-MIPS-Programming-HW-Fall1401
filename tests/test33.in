4
735052094
142709403
416595580
988625493