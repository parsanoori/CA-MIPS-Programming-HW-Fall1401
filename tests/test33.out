142709403
416595580
735052094
988625493